### ISSUE TITLE
cls command spacing as well as spacing for input not wrapping to next line, global env vars not highlighted

### ISSUE THREAD

**Issue by SOSANA** [cmderdev/cmder#1032]
I am having an issue with cls, it leaves spacing issues, you'll also notice that now global env vars aren't highlighted as they use to be example git status etc

Any suggestions on resolving these issues? out of the box use to work now it doesn't :-1: 
![temp](https://cloud.githubusercontent.com/assets/2634185/16745608/ccc19ce6-4773-11e6-8031-16b2f3328a6a.PNG)

**Comment by SOSANA** [comment]
@kodybrown @jdsumsion @glucas @Stanzilla @kohenkatz any suggestions?

**Comment by kohenkatz** [comment]
I'm not sure I understand your description of the issue. Can you do a fresh install of cmder and write down exactly the steps you take to reproduce this problem?

**Comment by SOSANA** [comment]
@kohenkatz 

There are various spacing issues when typing as text as it is not wrapping to the next line, the other issue was highlighting text via global env vars like git, npm, etc as its all currently white tex. Another issue was using the clear screen command ```cls``` as it leaves a line space at top of cmdr window. as shown in blow pictures. Also note the text being all white

inital render of console spacing is normal at top of window
![inital](https://cloud.githubusercontent.com/assets/2634185/16789570/014b4194-486c-11e6-83d3-0993b1c9a430.PNG)

once we type the clear screen cls command add's line spacing at top of window
![clslinespace](https://cloud.githubusercontent.com/assets/2634185/16789675/dc461d28-486c-11e6-9165-a5851a7d9876.PNG)

when start to type adds new line spacing instead of wrapping to next line
![wrappingnextline](https://cloud.githubusercontent.com/assets/2634185/16789751/689fc90e-486d-11e6-9c70-99443139fc57.PNG)

I simply down load your full version from your site, extract it, and place it in the directory of my choice. I than run the executable

**Comment by kohenkatz** [comment]
The issue here appears to have something to do with the Git prompt. I can reproduce it when I am inside a Git repo, but everything works normally when in a directory that is not a Git repo.

@SOSANA Can you confirm that it works properly in another directory that isn't a Git repository?

**Comment by kohenkatz** [comment]
I'm still not sure what you mean about the problem with the text color. As far as I know, the text you type is always supposed to be the same color. Only text that is printed after you hit `enter` gets colored.

**Comment by SOSANA** [comment]
@kohenkatz I can confirm I see the **same results** when not using git prompt

Prior to last few months of updates I used to get highlighted text in the command prompt for doing various commands. Unfortunately I don't have any prior screen shots to provide to show you an example. Maybe I installed at that time a third party program that did this text highlighting but that is my only other thing I can think of if this feature has never existed prior.

Any suggestions for quick hack to prevent these line spacings?
